### DISCUSSION TITLE
Where can I find more detailed information on the python commands?

### DISCUSSION THREAD

**Discussion by eric-bks** [openai/whisper#1493] [Q&A]
The README.md doesn't have enough info, and digging around the files confuses me.

Bonus points for Jupyter-specific documentation.

**Comment by phineas-pta** [comment]
wdym "python command" ? u want cli or use in python script ?

**Reply by eric-bks** [reply]
In python

**Reply by phineas-pta** [reply]
it's already in README

what does it miss
